spec/spec_helper.rb

```diff
@@ -15,6 +15,19 @@
     # `true` in RSpec 4.
     mocks.verify_partial_doubles = true
   end
+
+  config.before :each do
```

Comment:
Many of the specs change the global Devise.setup config settings so we pulled out before and after reset behavior into the spec helper.